### DISCUSSION TITLE
Education pack is asking about school email address but They don't provide any

### DISCUSSION THREAD

**Discussion by SpaceisGood** [community/community#47457] [GitHub Education]
Hello, I am trying to get Student Developer Pack and I have the school ID card in which they clearly mentioned the `Valid upto` date but they don't provide any email for the university, and Github rejected my proposal stating that "I should add an email address for the selected school" But I don't have any. I tried to upload two images of the card but I think I can't upload more than one image.

What should I do ?

**Comment by jge162** [comment]
Only GitHub support can help you. We can only discuss it here.
